src/mmrelay/migrate.py

```
@@ -0,0 +1,2497 @@
+
+            # Copy from backup (keep backup intact for safety)
+            if backup_path.is_dir():
+                shutil.copytree(str(backup_path), str(restore_path))
+            else:
+                shutil.copy2(str(backup_path), str(restore_path))
+
+            logger.info(
+                "Restored %s from backup %s to %s", step_name, backup_path, restore_path
+            )
+            return True
+        except (OSError, IOError, shutil.Error) as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)